<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_clickAca_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>319</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Opcion 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opcion 2 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opcion 3</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Numero 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numero1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Numero 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numero2"/>
     </item>
     <item>
      <widget class="QPushButton" name="boton_clickAca">
       <property name="text">
        <string>Sumar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultado"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>87</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="opcion1">
       <property name="text">
        <string>Suma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion2">
       <property name="text">
        <string>Multiplicacion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion3">
       <property name="text">
        <string>Divisi√≥n</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
